Index: src/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>public class Main {\r\n    public static void main(String[] args) {\r\n        System.out.println(\"Hello world!\");\r\n        System.out.println(\"A\");\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Main.java b/src/Main.java
--- a/src/Main.java	(revision 244ac2d0086ee64a3d983a082bcbd206ffacbbca)
+++ b/src/Main.java	(date 1694165952186)
@@ -2,5 +2,8 @@
     public static void main(String[] args) {
         System.out.println("Hello world!");
         System.out.println("A");
+        System.out.println("B");
+        System.out.println("B");
+        System.out.println("B");
     }
 }
\ No newline at end of file
